docs/Contributors.mill

```diff
@@ -0,0 +1,39 @@
+// See LICENSE for license details.
+
+package build.docs
+
+import github4s.Github
+import github4s.Github._
+import github4s.domain.User
```

Comment:
I was hoping git would detect these new `.mill` files (other than `docs/package.mill`) as moves with modification from the old `.scala` files in `project/` but it doesn't seem to.

Regardless, these are moves, feel free to review their code but just note they are very much existing code (again, other than `docs/package.mill`).